.github/workflows/package-release.yml

```diff
@@ -0,0 +1,64 @@
+#release.yml
+name: sgc-release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  sgcreleaser:
+    name: secretgen controller release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.6
+
+      - name: Install Carvel Tools
+        uses: vmware-tanzu/carvel-setup-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Setting up minikube
+        run: |
+          set -e -x
+          minikube start --driver=docker --wait=all
+          docker buildx create minikube --use --driver=kubernetes --bootstrap
+
+      - name: Fetch kctrl
+        uses: actions/checkout@v3
+        with:
+          ref: kctrl-pkg-init-release
```

Comment:
Currently, Pointing to the branch where we have the changes. Will update it once the changes got merged.